boa_tester/src/read.rs

```diff
@@ -186,7 +186,8 @@ fn read_metadata(code: &str) -> io::Result<MetaData> {
         .ok_or_else(|| io::Error::new(io::ErrorKind::InvalidData, "no metadata found"))?
         .get(1)
         .ok_or_else(|| io::Error::new(io::ErrorKind::InvalidData, "no metadata found"))?
-        .as_str();
-
-    serde_yaml::from_str(yaml).map_err(|e| io::Error::new(io::ErrorKind::InvalidData, e))
+        .as_str()
+        .replace('\r', "\n");
+    dbg!(&yaml);
```

Comment:
Was this just for debugging or is it meant to stay?

Reply:
Good catch :) removed!